src/lib.rs

```diff
@@ -2803,6 +2818,8 @@ impl Build {
                     format!("arm-kmc-eabi-{}", gnu)
                 } else if target.arch == "aarch64" && target.vendor == "kmc" {
                     format!("aarch64-kmc-elf-{}", gnu)
+                } else if target.os == "nto" {
+                    "qcc".into()
```

Comment:
Should this also support `q++`?

Reply:
My main concern was to compile the Rust toolchain without having to set a bunch of environment variables (which only builds C files), but I think it makes sense to add support for `q++`. Good point, thanks.

Reply:
Should be resolved.